### PR TITLE
[backport] support order argument in cupy.ones, cupy.full and cupy.eye

### PR DESCRIPTION
**Pull request by takagi**
Backport of #3655

### CODE REVIEW

**Comment by takagi** [comment]
[automatic post] Jenkins, test this please.

**Comment by emcastillo** [comment]
Seems like it is not working well 😅

**Comment by takagi** [comment]
I've fixed that, PTAL!

**Comment by leofang** [comment]
Jenkins, test this please

**Comment by leofang** [comment]
I think we may need to skip older numpy for `eye()`.

**Comment by leofang** [comment]
Jenkins, test this please

File: tests/cupy_tests/creation_tests/test_basic.py [hunk 1]
```diff
@@ -161,11 +161,19 @@ def test_empty_zero_sized_array_strides(self, order):
         b = cupy.empty((1, 0, 2), dtype='d', order=order)
         self.assertEqual(b.strides, a.strides)
 
+    @testing.for_CF_orders()
```
Comment (leofang) [inline tests/cupy_tests/creation_tests/test_basic.py:164]: This line needs to be removed 😅

Comment (takagi) [reply tests/cupy_tests/creation_tests/test_basic.py:164]: oh...

**Comment by leofang** [comment]
Jenkins, test this please

**Pull request merged**